kyverno-policy-reporter-ui.yaml

```diff
@@ -65,7 +72,30 @@ update:
     tag-filter-prefix: v
 
 test:
+  environment:
+    contents:
+      packages:
+        - curl
+    environment:
+      KO_DATA_PATH: "/var/run/ko"
   pipeline:
-    # AUTOGENERATED
     - runs: |
         policyreporter-ui --help
+    - name: Run and test ${{package.name}}
+      uses: test/daemon-check-output
+      with:
+        start: /usr/bin/policyreporter-ui run
+        timeout: 30
+        expected_output: |
+          register UI
+          /var/run/ko/ui
+          Server starts
+        # Can't use default error string set because of this log line `[ERROR] failed to load config file: open config.yaml: no such file or directory`.
```

Comment:
Let's `curl -sLO https://raw.githubusercontent.com/kyverno/policy-reporter-ui/refs/heads/main/backend/config.example.yaml` this in the `setup:` and try to omit the `error_strings`.

Reply:
thank you for the suggestion, I've adopted this approach.